app/javascript/channels/mentorRequestChannel.ts

```diff
@@ -0,0 +1,31 @@
+import consumer from '../utils/action-cable-consumer'
+import { MentorSessionRequest } from '../components/types'
+
+export type ChannelResponse = {
+  uuid: string
```

Comment:
we probably don't need the `uuid`, but it is probably always nice to have an identifier around.